README.md

```diff
@@ -19,7 +19,7 @@ Note: A corpnet account is required to view the VSTS build.
 
 ## Versions
 
-Current version - 1.13.1
+Current version - 1.13.2
 Minimum recommended version - 1.1.16
```

Comment:
do you know what is the basis of reccomending 1.1.16 as minimum?

Reply:
I don't know for now. Let me check the changelog.

Reply:
It should be because of the AndroidKeyStore bug hotfix according to the changelog. 
"Hot fix for catching RuntimeException thrown due to AndroidKeyStore bug, so App won't crash due to keystore got reset."
This is the only fix in 1.1.16. @addev-ashish

Reply:
ok, thanks for checking.